scripts/generate_cloudevent_library.dart

```diff
@@ -81,8 +88,12 @@ Future<void> generateLibraryFiles(
       final srcReferencePath =
           p.relative(normalizedPath, from: '$libFileDirectory/..');
 
-      libFiles[libraryFilePath]!
-          .writeln("export '$srcReferencePath' show ${symbols.join(', ')};");
+      // Dart export statements require forward slashes for paths, regardless of the
+      // operating system. This ensures compatibility, especially on Windows.
+      final dartSrcReferencePath =
+          srcReferencePath.replaceAll(p.separator, '/');
```

Comment:
I think that we can also write this as `srcReferencePath.replaceAll(p.separator, p.posix.separator);` what do you think about that instead?